radicle/src/cob/patch.rs

```diff
@@ -603,8 +603,14 @@ impl CodeComment {
 #[derive(Debug, Default, Clone, PartialEq, Eq)]
 pub struct Review {
     /// Review verdict.
+    ///
+    /// Nb. if the verdict is set and a subsequent review is made with
+    /// the verdict as `None`, the original verdict will be nullified.
     verdict: LWWReg<Option<Verdict>>,
     /// Review general comment.
+    ///
+    /// Nb. if the comment is set and a subsequent review is made with
+    /// the comment as `None`, the original comment will be nullified.
```

Comment:
Wouldn't this be expected behavior of LWWReg<Option<..>>?

Reply:
It is, but maybe it's not so obvious from an API perspective? I guess another approach would be to add this comment to the API function, ie. `patch.update`? :shrug:

Reply:
Or patch.review/comment?  I'd tend towards the API comment as well.  Otherwise maybe its a comment on LWWReg<Option<..>> just to avoid repetition.

Reply:
I think this is everywhere `LWWReg<Option<_>>` is used 😄 But I get yis, it does make sense to have it on the API, however, which part of the API? The `Action`? In the `Transaction`? Somewhere else?

I'll be able to update this when I'm fully back, otherwise happy for someone to change it to get the PR over the line :)